pkg/3scale/amp/component/system_monitoring.go

```diff
@@ -30,6 +30,75 @@ func (system *System) SystemSidekiqPodMonitor() *monitoringv1.PodMonitor {
 	}
 }
 
+func (system *System) SystemAppPodMonitor() *monitoringv1.PodMonitor {
+	return &monitoringv1.PodMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   "system-app",
+			Labels: system.Options.CommonSidekiqLabels,
+		},
+		Spec: monitoringv1.PodMonitorSpec{
+			PodMetricsEndpoints: []monitoringv1.PodMetricsEndpoint{
```

Comment:
Is there any difference between doing this or creating 3 PodMonitors with each monitor including only PodMetricsEndpoints to a specific container?

Keep in mind the selector of PodMonitor is of Pod Labels (containers do not have labels on their own if I'm not wrong)

Reply:
:+1: